test/benchmarks/material/checkbox/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("@npm_angular_dev_infra_private//benchmark/component_benchmark:component_benchmark.bzl", "component_benchmark")
```

Comment:
Cannot comment on the file above. What is `benchmark-driver.ts` doing?

Reply:
I think that file might have been outdated / deleted. Does that file still exist?

Reply:
Yeah, you might want to remove that file in this PR.